### ISSUE TITLE
Please add support for status.net accounts

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#114]
```
http://status.net StatusNet is the open source microblogging platform that 
helps you share and connect in real-time within your own domain.

It is developed by the same community that brings you http://identi.ca, in 
fact identi.ca is a status.net site. 
```

Original issue reported on code.google.com by `Alphak...@gmail.com` on 3 Mar 2010 at 8:53

**Comment by GoogleCodeExporter** [comment]
```
Do you mean the support of all status.net services?
```

Original comment by `tro...@gmail.com` on 3 Mar 2010 at 9:24
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Everything from x.status.net sites or personal statusnet sites.  See 
http://status.net/wiki/Development
```

Original comment by `Alphak...@gmail.com` on 3 Mar 2010 at 9:40

**Comment by GoogleCodeExporter** [comment]
```
I plan to add an "API proxy" for supporting all services with REST API.
```

Original comment by `tro...@gmail.com` on 3 Mar 2010 at 10:12
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by c90f9ba547 revision.
```

Original comment by `tro...@gmail.com` on 6 Mar 2010 at 1:34
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
That was very fast, Thanks! :) 
```

Original comment by `Alphak...@gmail.com` on 8 Mar 2010 at 1:40
